.buildkite/release-pipelines/start-code-freeze.yml

```diff
@@ -10,9 +10,13 @@ steps:
     # The first client to implement releases in CI was Android so the automation works in that queue.
     # We might want to move it to a leaner one in the future.
```

Comment:
Can this comment be removed now that it has been moved to a different agent?

Reply:
Thanks for noticing this!

[27f7fe6](https://github.com/Automattic/simplenote-ios/pull/1653/commits/27f7fe64ce81eaf75e8cacb687669a78bff80289)